openpype/hosts/aftereffects/api/lib.py

```diff
@@ -49,7 +49,10 @@ def get_unique_layer_name(layers, name):
             names[layer_name] = names[layer_name] + 1
         else:
             names[layer_name] = 1
-    occurrences = names.get(name, 0)
+    occurrences = names.get("{}{}".format(get_stub().LOADED_ICON, name), 0)
+
+    if occurrences !=0 and is_psd:
```

Comment:
Need a comment in the code to explain what is this code addition, this seems to be weird to do a division by two

Reply:
It's because in AE, loading a psd create a comp+a folder, it means 2 entities. So if there's one imported we'll have comp_v001, folder_v001
But if we import an other instance of the asset, since there's two element, they will be imported as comp_v00
 and folder_v003
 
 I'll had a comment to explain